static/usr/lib/systemd/system/fix-htctosys.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Work around automatic htctosys
```

Comment:
(nitpick) I would make this a bit more descriptive, maybe: `Work around time moving backwards when a rtc module is inserted and the kernel htctosys option is set` (or something like this, maybe this line is a bit too long).

Reply:
I am not sure this is the point of `Description` in [systemd.unit](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Description=). It says "this string should identify the unit rather than describe it".

If it failed to run it would say in the log: "Failed to start Work around time moving backwards when a rtc module is inserted and the kernel htctosys option is set". Which is a bit weird.

Reply:
Nitpick: isn't it hctosys instead of htctosys? And in fact the service name has the same issue.

Reply:
Right!

Reply:
Fixed.

Reply:
Thank you, I was actually not aware that description is meant this way.